literature.bib

```diff
@@ -9081,6 +9343,29 @@ @article{BSRC10
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2009 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{KAT:TOOLSeurope09,
+	author = "K{\"a}stner, Christian and Apel, Sven and Trujillo, Salvador and Kuhlemann, Martin and Batory, Don",
```

Comment:
use { and } instead of " everywhere

Reply:
Oops, that one slipped through. 😅 I also fixed other occurences of "" now.